requirements/requirements.txt

```diff
@@ -1,6 +1,5 @@
 appdirs==1.4.3
-autoenv==1.0.0
-bidict==0.13.1
+bidict==0.17.2
```

Comment:
Is `bidict` used in the codebase or a dependency for another library?

Reply:
I believe it was used in the past but currently is instructed to be used if needed on `line 349` of  `settings.py`

# TERSE_TO_LONG UNUSED.  IF NEEDED, change LONG_TO_TERSE to a bidict or invert the dict.

Reply:
If it isn't being used I recommend the package with original version (bidict==0.13.1) be moved to `legacy_requirements.txt` for now.

Reply:
Just saw this, `bidict` has now been moved to `requirements-legacy.txt`